src/main/java/teammates/common/datatransfer/questions/FeedbackMsqQuestionDetails.java

```diff
@@ -332,7 +414,19 @@ public String getQuestionSpecificEditFormHtml(int questionNumber) {
                 Slots.TEAM_SELECTED, generateOptionsFor == FeedbackParticipantType.TEAMS ? "selected" : "",
                 Slots.TEAMS_TO_STRING, FeedbackParticipantType.TEAMS.toString(),
                 Slots.INSTRUCTOR_SELECTED, generateOptionsFor == FeedbackParticipantType.INSTRUCTORS ? "selected" : "",
-                Slots.INSTRUCTORS_TO_STRING, FeedbackParticipantType.INSTRUCTORS.toString());
+                Slots.INSTRUCTORS_TO_STRING, FeedbackParticipantType.INSTRUCTORS.toString(),
+                Slots.MSQ_IS_MAX_SELECTABLE_CHOICES_ENABLED, isMaxSelectableChoicesDisabled ? "" : "checked",
+                Slots.MSQ_IS_MIN_SELECTABLE_CHOICES_ENABLED, isMinSelectableChoicesDisabled ? "" : "checked",
+                Slots.MSQ_PARAM_ENABLED_MAX_SELECTABLE_CHOICES,
+                        Const.ParamsNames.FEEDBACK_QUESTION_MSQ_ENABLE_MAX_SELECTABLE_CHOICES,
+                Slots.MSQ_PARAM_ENABLED_MIN_SELECTABLE_CHOICES,
+                        Const.ParamsNames.FEEDBACK_QUESTION_MSQ_ENABLE_MIN_SELECTABLE_CHOICES,
+                Slots.MSQ_PARAM_MAX_SELECTABLE_CHOICES, Const.ParamsNames.FEEDBACK_QUESTION_MSQ_MAX_SELECTABLE_CHOICES,
+                Slots.MSQ_PARAM_MIN_SELECTABLE_CHOICES, Const.ParamsNames.FEEDBACK_QUESTION_MSQ_MIN_SELECTABLE_CHOICES,
+                Slots.MSQ_MAX_SELECTABLE_CHOICES,
+                        isMaxSelectableChoicesDisabled ? "2" : Integer.toString(maxSelectableChoices),
```

Comment:
Query: what is the meaning of value "2" here?

Reply:
2 is the default value to be shown. It is also the least permitted value for `maxSelectableChoices`, because if it were 1, the instructor would be better off creating a MCQ question instead.

Should I create a constant for this? If yes, should I place it in `FeedbackMsqQuestionDetails` itself or in `Const.java`?

Reply:
Understood. It should be fine to be let as magic number for now.